client/blocks/inline-help/contextual-help.js

```diff
@@ -52,700 +59,795 @@ const contextLinksForSection = {
+			),
 		},
 	],
 	'settings-discussion': [
 		{
 			link: 'http://en.support.wordpress.com/settings/discussion-settings/',
 			post_id: 1504,
-			title: 'Discussion Settings',
-			description:
-				'The Discussion Settings are used to control how visitors and other blogs interact with your site.',
+			title: translate( 'Discussion Settings' ),
+			description: translate(
+				'The Discussion Settings are used to control how visitors and other blogs interact with your site.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/enable-disable-comments-for-future-posts/',
 			post_id: 5997,
-			title: 'Enable and Disable Comments for Future Posts',
-			description:
-				'You can enable/disable comments on future posts by going into your Discussion settings. ',
+			title: translate( 'Enable and Disable Comments for Future Posts' ),
+			description: translate(
+				'You can enable/disable comments on future posts by going into your Discussion settings. '
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/comments/',
 			post_id: 113,
-			title: 'Comments',
-			description: 'Comments are a way for visitors to add feedback to your posts and pages.',
+			title: translate( 'Comments' ),
+			description: translate(
+				'Comments are a way for visitors to add feedback to your posts and pages.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/subscriptions-and-newsletters/',
 			post_id: 67810,
-			title: 'Subscriptions and Newsletters',
-			description:
-				'Learn how readers can subscribe to your blog to receive email notifications of all of your posts.',
+			title: translate( 'Subscriptions and Newsletters' ),
+			description: translate(
+				'Learn how readers can subscribe to your blog to receive email notifications of all of your posts.'
+			),
 		},
 	],
 	'settings-traffic': [
 		{
 			link: 'http://en.support.wordpress.com/getting-more-views-and-traffic/',
-			title: 'Get More Views and Traffic',
-			description:
-				'Want more traffic? Here are some tips for attracting more visitors to your site!',
+			title: translate( 'Get More Views and Traffic' ),
+			description: translate(
+				'Want more traffic? Here are some tips for attracting more visitors to your site!'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/related-posts/',
 			post_id: 1545,
-			title: 'Related Posts',
-			description:
-				'The Related Posts feature pulls relevant content from your blog to display at the bottom of your posts.',
+			title: translate( 'Related Posts' ),
+			description: translate(
+				'The Related Posts feature pulls relevant content from your blog to display at the bottom of your posts.'
+			),
 		},
 		{
 			link: 'http://en.support.wordpress.com/webmaster-tools/',
 			post_id: 5022,
-			title: 'Webmaster Tools',
-			description:
-				'Learn how to verify your WordPress.com site for the webmaster tools that many search engines provide.',
+			title: translate( 'Webmaster Tools' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( 'Webmaster Tools Verification' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).